File: PageObjectDotNet/PHPTravel/PageObjects/FeaturedCar.cs
using OpenQA.Selenium;
using OpenQA.Selenium.Support.PageObjects;
using SeleniumFramework.Base;
using System.Collections.Generic;
using System.Globalization;

namespace PHPTravel.PageObjects
{
    public class FeaturedCar : BasePage
    {
        [FindsBy(How = How.CssSelector, Using = "i.fa-star")]
        private IList<IWebElement> rate;

        [FindsBy(How = How.CssSelector, Using = "span.text-center")]
        private IWebElement currencyAndPrice;

        [FindsBy(How = How.CssSelector, Using = "h4.ellipsis")]
        private IWebElement name;

        [FindsBy(How = How.CssSelector, Using = "div.wow p")]
        private IWebElement location;


        public int GetRate()
        {
            return rate.Count;
        }

        public string GetCarName()
        {
            return Support.GetTextUsingJavaScript(Driver, name);
        }

        public float GetCarPrice()
        {
            var txt = currencyAndPrice.Text.Split('$')[1];
            return float.Parse(txt, NumberStyles.AllowThousands);
        }

        public string GetCurrency()
        {
            return currencyAndPrice.FindElement(By.TagName("small")).Text;
        }

        public string GetLocation()
        {
            return Support.GetTextUsingJavaScript(Driver, location);
        }

    }
}

File: PageObjectJava/src/test/java/PageObjects/FeaturedCar.java
package PageObjects;

import Base.BasePageObject;
import Support.Support;
import org.openqa.selenium.By;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.support.FindBy;

import java.util.List;

public class FeaturedCar extends BasePageObject {

    @FindBy(css = "i.fa-star")
    private List<WebElement> rate;

    @FindBy(css = "span.text-center")
    private WebElement currencyAndPrice;

    @FindBy(tagName = "h4")
    private WebElement name;

    @FindBy(css = "div.wow p")
    private WebElement location;


    public int getRate()
    {
        return rate.size();
    }

    public String getCarName()
    {
        return Support.extractTextWithJavaScript(name);
    }

    public float getCarPrice()
    {
        String txt = currencyAndPrice.getText();
        txt = txt.substring(txt.indexOf(" ") + 2);
        return Float.parseFloat(txt);
    }

    public String getCurrency()
    {
        return currencyAndPrice.findElement(By.tagName("small")).getText();
    }

    public String getLocation()
    {
        return Support.extractTextWithJavaScript(location);
    }

}

